default/scripting/species/common/telepathic.macros

```diff
@@ -0,0 +1,16 @@
+TELEPATHIC_DETECTION
+'''     EffectsGroup
+            description = "TELEPATHIC_DETECTION"
+            scope = And [
+                PopulationCenter
+                Not OR [
+                    Species name = Source.Species
+                    Species name = "SP_EXOBOT"
```

Comment:
Flavour wise, I'm happy with that as is, Trith learn to mask their thoughts, other telepaths don't have to/need to. Balance wise, the effect gives visibility of the planets that are causing the population malus, so it makes sense as is.  That's not to say it should be as is, merely that for initial testing *and* for flavour this makes the most sense.